src/configure/configure.ts

```diff
@@ -513,9 +525,12 @@ class Orchestrator {
         if (!repoAnalysisResult || !repoAnalysisResult.applicationSettingsList) {
             return;
         }
+        let workingDirectories = Array.from(new Set(this.inputs.potentialTemplates
+            .filter((template) => template.templateType === TemplateType.REMOTE)
+            .map((template: RemotePipelineTemplate) => template.workingDirectory)));
         var applicationSettings = repoAnalysisResult.applicationSettingsList.filter(applicationSetting => {
             if (this.inputs.pipelineConfiguration.template.templateType === TemplateType.REMOTE) {
-                return true;
+                return workingDirectories.indexOf(applicationSetting.settings.workingDirectory) > 0;
```

Comment:
Would it be better to handle casing here. Although not required currently, it is better to not take dependency on such facts.

Reply:
done.